GoogleForCreators/web-stories-wp
# Redesign: Video accessibility panel

## Summary
- Removes Poster panel as a separate panel.
- Uses the redesigned TextArea for the poster alt.


## To-do



## User-facing changes
| Before | After |
|-----|---------|
| <img width="336" alt="Screenshot 2021-03-10 at 15 19 42" src="https://user-images.githubusercontent.com/3294597/110635702-ae624a80-81ab-11eb-81a7-d3b029ede645.png"> | <img width="320" alt="Screenshot 2021-03-09 at 17 51 16" src="https://user-images.githubusercontent.com/3294597/110635590-8d015e80-81ab-11eb-9754-240c48f59b46.png"> |



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a video
2. Open the Video Accessibility panel and verify it looks as [in Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=742%3A22145) -- note that the Poster Title field is removed purposefully, the design needs an update.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6466
